checks/checkforfile.go

```diff
@@ -22,7 +22,11 @@ import (
 // for the occurrence.
 func CheckIfFileExists(checkName string, c *checker.CheckRequest, onFile func(name string,
```

Comment:
CheckIfFileExists is no longer used. I think you can remove it as part of this PR.

Reply:
Done.